### PR TITLE
255 report form save for user

### PR DESCRIPTION
**Pull request by Xapp73**

### CODE REVIEW

**Comment by Xapp73** [comment]
[Report form save for user](https://trello.com/c/yws2949H/255-report-form-save-for-user)

**Comment by Xapp73** [comment]
[Update UserModule and RBAC to use configurable user entity instead of hardcoded one. Then move user entity to the core.](https://trello.com/c/TtgMRzTG/298-update-usermodule-and-rbac-to-use-configurable-user-entity-instead-of-hardcoded-one-then-move-user-entity-to-the-core)

File: src/Controller/UserController.php [hunk 1]
```diff
@@ -167,7 +173,8 @@ public function editAction()
             return;
         }
 
-        $user = $this->entityManager->getRepository(User::class)
+        /** @var UserInterface $user */
+        $user = $this->entityManager->getRepository($this->userEntityClassName)
```
Comment (HesyRa) [inline src/Controller/UserController.php]: Use function `getUserById` from this class

File: src/Controller/UserController.php [hunk 2]
```diff
@@ -143,7 +149,7 @@ public function viewAction()
         }
 
         // Find a user with such ID.
-        $user = $this->entityManager->getRepository(User::class)
+        $user = $this->entityManager->getRepository($this->userEntityClassName)
```
Comment (HesyRa) [inline src/Controller/UserController.php]: Use function `getUserById` from this class

**Review by HesyRa — CHANGES_REQUESTED** [review]

**Review by ikovalyov — APPROVED** [review]

**Pull request merged**